samples/fr/account_type_not_supported.json

```diff
@@ -0,0 +1,156 @@
+      }
+    },
+    {
+      "transactions": [
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-06T12:00:00.000Z"
+          },
+          "description": "PRELEVEMENT HARRY POTTER PRELEVEMENT CARTE ZERO FAST3280420323",
+          "amount": -101.67
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-06T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA POTTER HARRY ECH ID EMETTEUR/LU66ZZZ000023 MDT/3BEAC43A69zeeeeAC6 REF/1B0F18BC115A4788DA4 LIB/PRELEVEMENT CARTE ZERO FSSI043def106",
+          "amount": -150
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRELEVEMENT DE NAVIGO ANNUEL - G COMUTITRES REF : 3/262663-1/92921 Navigo Annuel",
+          "amount": -75.2
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA CREDIPAR Motif : 101M3323222 32 VOTRE IDEN",
+          "amount": -399.89
+        }
+      ],
+      "balance": -120,
+      "balanceDate": "2023-08-31T12:00:00.000Z",
+      "iban": "FR7158308147881N2Z8VVNBLU24",
+      "currency": "EUR",
+      "type": "UNKNOWN",
+      "usage": "PERSONAL",
+      "name": "Compte carte 01",
+      "bank": {
+        "name": "Algoan Demo",
+        "id": "AD",
+        "logoUrl": "https://storage.googleapis.com/bank-data-images-preprod/algoan-bank/logo.svg"
+      }
+    },
+    {
+      "transactions": [
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-06T12:00:00.000Z"
+          },
+          "description": "PRELEVEMENT HARRY POTTER PRELEVEMENT CARTE ZERO FAST3280420323",
+          "amount": -101.67
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-06T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA POTTER HARRY ECH ID EMETTEUR/LU66ZZZ000023 MDT/3BEAC43A69zeeeeAC6 REF/1B0F18BC115A4788DA4 LIB/PRELEVEMENT CARTE ZERO FSSI043def106",
+          "amount": -150
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRELEVEMENT DE NAVIGO ANNUEL - G COMUTITRES REF : 3/262663-1/92921 Navigo Annuel",
+          "amount": -75.2
+        },
+        {
+          "currency": "EUR",
+          "dates": {
+            "debitedAt": "2024-04-07T12:00:00.000Z"
+          },
+          "description": "PRLV SEPA CREDIPAR Motif : 101M3323222 32 VOTRE IDEN",
+          "amount": -399.89
+        }
+      ],
+      "balance": 2200,
+      "balanceDate": "2023-08-31T12:00:00.000Z",
+      "iban": "FR195226475633NGNDOGOL7KZ15",
```

Comment:
You use the same IBAN for ownership not supported. Could you use a different one please?